crates/voicevox_core_java_api/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "voicevox_core_java_api"
+version.workspace = true
+edition.workspace = true
+publish.workspace = true
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+android_logger = "0.13.1"
+anyhow.workspace = true
+chrono = "0.4.26"
+easy-ext.workspace = true
+jni = "0.21.1"
+once_cell.workspace = true
+serde_json.workspace = true
+tokio.workspace = true
+tracing = { workspace = true, features = ["log"] }
```

`tracing/log`はもう`workspace.dependencies`に含まれているため:

```suggestion
tracing.workspace = true
```